pages/glossary/semantic-widget.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Semantic widget
+key: semantic-widget
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - DOM tree
+---
+
+A _semantic widget_ is any element whose [semantic role][] is one of the [widget roles][], that is a non-[abstract role][] that inherits from the [`widget`][widget] role.
```

Comment:
I'm not sure we need the second part of the sentence (i.e. after the comma). I don't see the added value in explaining what are "widget roles".
What I would consider adding is a clarification that both "standalone user interface widgets" and "composite user interface widgets" are covered by the definition.

Reply:
Wasn't sure either and went through several iterations. The initial train of thought was:
* Copy semantic link => "role of widget or inheriting from widget".
* But `widget` shouldn't be used as role, so remove it.
* But it looks weird that `widget` is not a "semantic widget", this need clarification.
* Oh wait, there's actually a list already.

🤷 

Maybe it is better now 😁